spec/features/visitors/sign_up_spec.rb

```diff
@@ -195,15 +195,27 @@
     expect(page).to have_css('input.password[type="text"]')
   end
 
-  scenario 'visitor is redirected back to password form when password is invalid' do
-    create(:user, :unconfirmed)
-    confirm_last_user
-    fill_in 'password_form_password', with: 'Q!2e'
+  context 'password is invalid' do
+    scenario 'visitor is redirected back to password form' do
+      create(:user, :unconfirmed)
+      confirm_last_user
+      fill_in 'password_form_password', with: 'Q!2e'
 
-    click_button t('forms.buttons.submit.default')
+      click_button t('forms.buttons.submit.default')
```

Comment:
if I did it wasn't intentional. I don't see extra space in the file now. Maybe I was "fixing" it?